config/application.rb

```diff
@@ -115,12 +115,12 @@ def debug?
 
     # @return [Boolean]
     def gov_one_login?
-      Types::Params::Bool[ENV.fetch('GOV_ONE_LOGIN', false)]
+      Types::Params::Bool[ENV.fetch('GOV_ONE_LOGIN', true)] && !live?
```

Comment:
So after this feature is merged, enabled in staging, but off in prod; what's the workflow going to be if we have 20 subsequent versions before we are ready to enable the feature?